mllib/src/main/scala/org/apache/spark/mllib/feature/StandardScaler.scala

```diff
@@ -61,20 +61,34 @@ class StandardScaler(withMean: Boolean, withStd: Boolean) extends Logging {
  * :: Experimental ::
  * Represents a StandardScaler model that can transform vectors.
  *
- * @param withMean whether to center the data before scaling
- * @param withStd whether to scale the data to have unit standard deviation
  * @param mean column mean values
  * @param variance column variance values
+ * @param withMean whether to center the data before scaling
+ * @param withStd whether to scale the data to have unit standard deviation
  */
 @Experimental
-class StandardScalerModel private[mllib] (
-    val withMean: Boolean,
-    val withStd: Boolean,
+class StandardScalerModel (
     val mean: Vector,
-    val variance: Vector) extends VectorTransformer {
+    val variance: Vector,
+    var withMean: Boolean,
+    var withStd: Boolean) extends VectorTransformer {
 
   require(mean.size == variance.size)
 
+  def this(mean: Vector, variance: Vector) {
+    this(mean, variance, false, true)
+  }
+
```

Comment:
@mengxr Just to make sure I'm clear, are you suggesting changing the `StandardScalerModel` to take the standard deviation vector (instead of variance)?  Or are you just calling it 'std' for short?